notebooks/discard-analysis-2018.ipynb

```diff
@@ -27,7 +27,7 @@
     "\n",
     "[disco]: https://www.measurementlab.net/blog/disco-dataset/\n",
     "[micro]: https://www.measurementlab.net/blog/traffic-microbursts-and-their-effect-on-internet-measurement/\n",
-    "[parser]: https://github.com/m-lab/etl/blob/integration/parser/disco.go"
+    "[parser]: https://github.com/m-lab/etl/blob/master/parser/disco.go"
```

Comment:
This link was throwing a 404 when attempting to commit the relevant changes for this PR, so I added a link to `/parser/disco.go` in the master branch. It appears that the `integration` branch has been removed since the switch discard post went live.